ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/PullQueryExecutor.java

```diff
@@ -317,6 +317,63 @@ TableRowsEntity queryRowsLocally(
     );
   }
 
+  @VisibleForTesting
+  TableRowsEntity forwardTo(
```

Comment:
since this got moved, it was hard to see what changed. but seems like you are just adding the flag.

Reply:
Yes, just added the flag